view/theme/frio/templates/field_yesno.tpl

```diff
@@ -1,21 +0,0 @@
-
-	<div class="field yesno">
-		<label for="id_{{$field.0}}">{{$field.1}}</label>
-		<div class="onoff toggle btn btn-xs pull-right" id="id_{{$field.0}}_onoff">
-			
-			<div class="switchlabel toggle-group">
```

Comment:
Please check none of the removed classes are specific to this template and will leave orphan CSS.